src/book/04-general/01-git.mdx

```diff
@@ -0,0 +1,59 @@
+---
+section: Guides
+chapter: General
+title: Git
+description: Contributing to a NUbots project with git.
+slug: /guides/general/git
+---
+
+## Prerequisites
+
+This guide assumes you know the basics of git and basic shell commands.
```

Comment:
Maybe specify what is basic.

Reply:
I have added which git commands are prerequisite, but I am not sure what to say about shell. It's mostly that you just can't be shellphobic lol